dump.go

```diff
@@ -51,6 +51,8 @@ func DumpToWriter(slice interface{}, writer io.Writer, options ...CsvOptions) er
 		csvWriter.Comma = option.Separator
 	}
 
+	csvWriter.UseCRLF = option.UseCRLF
```

Comment:
Don't we need a default to be backward compatible?

Reply:
UseCRLF is boolean value, it is backward compatible by default.